api/src/main/java/com/lovely4k/backend/member/service/MemberService.java

```diff
@@ -23,22 +29,38 @@ public MemberProfileGetResponse findMemberProfile(Long memberId) {
         return MemberProfileGetResponse.of(findMember);
     }
 
-    // 프로필 사진 수정
     @Transactional
-    public void updateMemberProfile(MemberProfileEditServiceRequest serviceRequest, Long memberId) {
+    public void updateMemberProfile(List<MultipartFile> profileImage, MemberProfileEditServiceRequest serviceRequest, Long memberId) {
```

Comment:
컬렉션은 복수로 표현하는 관습이 있습니다 ㅎㅎ
```suggestion
    public void updateMemberProfile(List<MultipartFile> profileImages, MemberProfileEditServiceRequest serviceRequest, Long memberId) {
```

Reply:
아하 이 부분을 놓치고있었네요 멘토님 감사합니당